pyproject.toml

```diff
@@ -40,7 +40,7 @@ dependencies = [
     "z3-solver>=4.11,<4.14",
     "qecsim",
     "ldpc>=0.1.50",
-    "numpy",
+    "numpy < 2",
```

Comment:
```suggestion
    "numpy < 2",  # some of our dependencies are not yet compatible with numpy 2.0
```

Reply:
Good point, thanks for adding the comment. I know it's a quick fix for now. But without this, the builds in my other PR fail.